.kodiak/config.yaml

```diff
@@ -0,0 +1,28 @@
+version: 1.0
+
+snow:
+  - id: 546046 # Milo Bacom (business.adobe.com)
+
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
+        assignee:
+          name: slavin
```

Comment:
@JasonHowellSlavin Should the tickets be assigned to Malik instead?

Reply:
Let's discuss at standup.